stock_location_orderpoint_cleanup/models/stock_move.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockMove(models.Model):
+
+    _inherit = "stock.move"
```

Comment:
Why are you creating an empty stock.move class?

Reply:
Hi, that's currently in progress (PR is draft ;-))

Reply:
Ooh, Oopsie 🙈